routes/segments_routes.py

```
@@ -1,76 +1,190 @@
-        return "info_db_no와 origin_table 파라미터가 필요합니다.", 400
+    if info_db_no is None or not user_info or not user_sub_info or not target_column:
+        return jsonify({"success": False, "message": "info_db_no, user_info, user_sub_info, target_column 파라미터가 필요합니다."}), 400
 
     try:
-        df = load_data(info_db_no, origin_table)
-        if df.empty:
-            return "해당 테이블에 데이터가 없습니다.", 404
-        df = add_segments(df)
+        df_user = load_data(info_db_no, user_info, user_sub_info, target_column)
+        df_sub = load_data(info_db_no, user_sub_info, user_info, target_column)
     except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)